Zend/zend_vm_def.h

```diff
@@ -1683,6 +1683,49 @@ ZEND_VM_HANDLER(136, ZEND_ECHO, CONST|TMPVAR|CV, ANY)
 	ZEND_VM_NEXT_OPCODE_CHECK_EXCEPTION();
 }
 
+// This is copied from ZEND_ECHO above with only a small part different
+ZEND_VM_HANDLER(200, ZEND_ECHO_ESCAPE, CONST|TMPVAR|CV, ANY)
```

Comment:
I had to update this from 199 -> 200 cause 199 is now used. I could pick a bigger number to help avoid future conflicts?

Reply:
What is this number used for? Is it basically an op code? 

I found this list: https://github.com/php/php-src/blob/master/Zend/zend_vm_handlers.h

which would indicated 200 is also used?

Reply:
Different set of numbers.